runtime/runtime/src/actions.rs

```diff
@@ -297,6 +311,9 @@ pub(crate) fn action_add_key(
         return;
     }
     set_access_key(state_update, account_id, &add_key.public_key, &add_key.access_key);
+    account.storage_usage += (add_key.public_key.as_ref().len()
```

Comment:
AccessKeys also use `account_id` and a separator as a prefix. Should we include them or we ignore them, because they are shared?

Reply:
We don't include them for contract data, so not doing it here either.

Reply:
>We don't include them for contract data, so not doing it here either.

You mean account data? Account storage cost has a fixed storage value, which works because it is one. We can have multiple access keys prefixed with very long account names. I suggest we add two fields to `RuntimeConfig` : storage usage of account in bytes (computed assuming Account is populated to the max) and base storage usage of the access key (computed assuming max account id length).

Reply:
Account data / data stored by contract executions. All of it it stored in the trie prefixed with account name, but we don't count the prefix.

Reply:
We then should have a base cost for having an entry in KV storage, both for account data and access keys. Because someone can create a ton of empty KVs (Let's say all combinations of 4 bytes per key) pay almost nothing but force us to use a ton of storage because of prefixes.